### PR TITLE
added php 7.2 && nightly in travis

### PR DESCRIPTION
**Pull request by kokspflanze**
added php 7.2 && nightly in travis
removed hhvm in travis

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -57,17 +57,27 @@ matrix:
...
         - DEPS=lowest
-    - php: hhvm 
+    - php: 7.2
       env:
         - DEPS=locked
-    - php: hhvm 
+    - php: 7.2
+      env:
+        - DEPS=latest
+    - php: nightly
```
Comment (webimpress) [inline .travis.yml]: Not sure if we need both - PHP 7.2 and nightly.

Comment (kokspflanze) [inline .travis.yml]: nightly is php7.3

Comment (webimpress) [inline .travis.yml]: Great, thanks! We should then update maintainers template probably ;-)

**Comment by froschdesign** [comment]
LGTM 👍

File: .travis.yml [hunk 2]
```diff
@@ -45,6 +30,8 @@ matrix:
     - php: 7
       env:
         - DEPS=locked
+        - CHECK_CS=true
```
Comment (webimpress) [inline .travis.yml]: Expected variable is `CS_CHECK`, please see line 84

File: .travis.yml [hunk 3]
```diff
@@ -57,25 +44,31 @@ matrix:
...
       env:
         - DEPS=latest
   allow_failures:
-    - php: hhvm
-
-notifications:
-  irc: "irc.freenode.org#zftalk.dev"
-  email: false
+    - php: 7.2
+    - php: nightly
```
Comment (webimpress) [inline .travis.yml]: Too early... Can be removed, see: https://github.com/zendframework/zend-expressive-hal/pull/9

File: .travis.yml [hunk 4]
```diff
@@ -57,25 +44,21 @@ matrix:
...
+    - php: 7.2
       env:
         - DEPS=latest
   allow_failures:
-    - php: hhvm
-
-notifications:
-  irc: "irc.freenode.org#zftalk.dev"
-  email: false
+    - php: 7.2
```
Comment (snapshotpl) [inline .travis.yml]: Now shouldn't be allowed to fail, right? :)

Comment (kokspflanze) [reply .travis.yml]: yes, its a pr from august 2017
i dont want to update it each month and wait more and more

**Comment by weierophinney** [comment]
Let's do the following:

- Allow 7.2 to pass
- Remove nightly (we can re-enable when either an alpha or beta is first released for 7.3)

**Comment by kokspflanze** [comment]
nightly was already removed with https://github.com/zendframework/zend-view/pull/135/commits/94e9ecccf084f0060a8684024c379a4f53caa79b

**Comment by weierophinney** [comment]
Thanks, @kokspflanze; merged to develop for release with 2.10.0.

**Pull request closed without merging**